_episodes/01-numpy.md

```diff
@@ -80,10 +125,18 @@ weight in pounds: 132.0
 ~~~
 {: .output}
 
-As the example above shows,
-we can print several things at once by separating them with commas.
+The above command, however, did not change the value of `weight_kg`:
+~~~
+print(weight_kg)
+~~~
+{: .language-python}
+
+~~~
+60
+~~~
+{: .output}
 
-We can also change a variable's value by assigning it a new one:
+To change variable's value, we have to assign it a new one:
```

Comment:
Depending on the type of variable (immutable vs. mutable), it is not true that "we have to assign it a new one" to change the value. I find the old wording more accurate. If you want to use the new wording, it should say "a variable's value".

Reply:
Hm.... interesting point. How about
```
To change value of `weight_kg` variable, we have to assign it a new one:
```

>  If you want to use the new wording, it should say "a variable's value".

I believe this contradicts the first statement:

> Depending on the type of variable (immutable vs. mutable), it is not true that "we have to assign it a new one